js/package.dist.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "composio-core",
+  "version": "0.4.4-beta",
+  "description": "",
+  "main": "index.js",
+  "scripts": {},
```

Comment:
The package.dist.json contains empty objects for 'scripts' and 'lint-staged'. These should either be populated with necessary configurations or removed to keep the distribution package clean and minimal.